test/unit/controllers/PeerSelectControllerSpec.js

```diff
@@ -2,22 +2,20 @@
 /* global describe, it, inject, expect, beforeEach */
 
 describe('PeerSelectController', function () {
-  var $controller, $scope, $q, $mod, $APManager, $EService, createController, timeoutTime, $promiseData, $promise, $promiseFlag
-
   beforeEach(module('myApp.controllers'))
 
   beforeEach(function () {
     // The modalInstance will propably usually give a boolean as return.
     // However, for testing purposes it is important to gain knowledge about the input of the function
-    $mod = {
+    this.$modalInstance = {
       close: function (arr) {
         return arr
       }
     }
 
-    timeoutTime = 1000
+    this.waitOneSecond = 1000
```

Comment:
[Number 28](https://opensource.com/article/17/5/30-best-practices-software-development-and-testing) resp.

![Debugging front-end by CommitStrip](https://www.commitstrip.com/wp-content/uploads/2017/05/Strip-Debugging-Front-english650-final.jpg)